aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/AWSCognitoAuthPlugin.java

```diff
@@ -426,6 +418,19 @@ public void onError(Exception exception) {
             }
         });
 
+        AWSMobileClient.getInstance().getUserAttributes(new Callback<Map<String, String>>() {
+            @Override
+            public void onResult(Map<String, String> result) {
+                sessionBuilder.userSub(result.get("sub"));
+                latch.countDown();
+            }
+
+            @Override
+            public void onError(Exception exception) {
```

Comment:
What happens to the content of this error? It looks ignored. How does it get back to the consumer, for auditing/reporting?

Reply:
Currently, we are ignoring it because the most common scenario for a failure is that the user attributes aren't available for the given user type but we still want to return the rest of the information for this session object which is available. However, it would be nice to also return errors for optional inspection so I'll be talking with JIthin and the Architecture team about this as an enhancement in a follow-up PR.

Reply:
Can you add this whole thing into an `/** Why we're ignoring this error ... */` comment?